packages/fxa-auth-server/test/remote/oauth_tests.js

```diff
@@ -191,6 +191,7 @@ describe('/oauth/ routes', function() {
       refresh_token: res.refresh_token,
     });
     assert.ok(res2.access_token);
+    assert.notOk(res2.id_token);
     // NOTE - this assertion is commented out because of
     // https://github.com/mozilla/fxa/issues/1867
     //assert.equal(res2.scope, SCOPE);
```

Comment:
what can we say about `res2.scope` here, empty?

Reply:
thanks, updated; it should contain just the "oldsync" scope, which is what will be on the issued token.